src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -1324,11 +1324,6 @@ private void parseItem(QuestionDef q, Element e) {
                 }
 
                 if (value != null) {
-                    if (value.length() > MAX_VALUE_LEN) {

```

Comment:
Can't you also remove the constant's declaration?

Reply:
Jenny, you know so much stuff